cmd/minishift/cmd/start.go

```diff
@@ -126,6 +127,7 @@ func runStart(cmd *cobra.Command, args []string) {
 	setDockerProxy()
 	setOcProxy()
 	setShellProxy()
```

Comment:
sounds good to me. Definitely not a super important feature, but kind of nice. I think I will remove the documentation though. This will give us more flexibility in case we want to remove it  later.